gradle-server/src/main/java/com/github/badsyntax/gradle/utils/Utils.java

```diff
@@ -39,4 +40,31 @@ private static Version getLowestGradleVersion(Version javaVersion) {
 		}
 		return new Version("2.0");
 	}
+
+	public static String normalizePackageName(String name) {
```

Comment:
see reference: https://github.com/gradle/gradle/blob/5ec3f672ed600a86280be490395d70b7bc634862/subprojects/build-init/src/main/java/org/gradle/buildinit/plugins/internal/PackageNameBuilder.java#L20

We can't directly use this util method since it will bring a dependency on Gradle itself. It will increase 150MB size into the current package.

Reply:
makes sense.